dataplane/linux/int_dataplane.go

```diff
@@ -288,76 +291,67 @@ type InternalDataplane struct {
 const (
 	healthName     = "int_dataplane"
 	healthInterval = 10 * time.Second
+
+	ipipMTUOverhead      = 20
+	vxlanMTUOverhead     = 50
+	wireguardMTUOverhead = 60
+	aksMTUOverhead       = 100
 )
 
-func NewIntDataplaneDriver(config Config) *InternalDataplane {
-	log.WithField("config", config).Info("Creating internal dataplane driver.")
-	ruleRenderer := config.RuleRendererOverride
+func NewIntDataplaneDriver(c Config) *InternalDataplane {
```

Comment:
Not sure about the global rename.  Presumably that was to avoid a clash with the imported name?  I'd probably solve that by moving the new function to a new file. Then we can shrink this god file a bit too.

I can live with the rename but it feels like a pretty large scope to have a variable with such a short name.  It might conflict when you merge it over to the private codebase.

Reply:
Correct, this was due to needing to reference the `config` package inside that function, but since I extracted a function for this functionality I was able to revert this rename.